easybuild/easyconfigs/f/FLANN/FLANN-1.8.4-foss-2020a-Python-3.8.2.eb

```diff
@@ -0,0 +1,46 @@
+# Contribution from the NIHR Biomedical Research Centre
+# Guy's and St Thomas' NHS Foundation Trust and King's College London
+# uploaded by J. Sassmannshausen
+
+easyblock = 'CMakeMake'
+
+name = 'FLANN'
+version = '1.8.4'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/mariusmuja/flann/'
+description = "FLANN is a library for performing fast approximate nearest neighbor searches in high dimensional spaces."
+
+toolchain = {'name': 'foss', 'version': '2020a'}
+toolchainopts = {'openmp': True}
+
+source_urls = ['https://github.com/mariusmuja/flann/archive/']
+sources = ['%(version)s.tar.gz']
+checksums = [
+    'ed5843113150b3d6bc4c325fecb51337838a9fc09ad64bdb6aea79d6e610ee13',  # flann-1.8.4.tar.gz
+    '491b1ead0bd4167d1d3e59cecb188cc54cb1ee854179b9571a68ab02f934bcb6',  # cmake-flann-1.8.4.patch'
+]
+
+builddependencies = [('CMake', '3.16.4')]
+dependencies = [
+    ('Python', '3.8.2'),
+    ('SciPy-bundle', '2020.03', '-Python-3.8.2'),
+]
+
+separate_build_dir = True
+
+patches = ['cmake-flann-1.8.4.patch']
+
+preconfigopts = 'touch %(builddir)s/flann-%(version)s/src/cpp/empty.cpp && '
```

Comment:
@sassy-crick it' cleaner if the empty file is added in the patch itself. you can do that for example as follows:
```
git diff flann-1.8.4.orig flann-1.8.4
```